src/components/Artists/Artists.test.js

```diff
@@ -1,9 +1,18 @@
 import React from "react";
 import Artists from "./ArtistsContainer";
+import {
+  shallow
+} from "enzyme"
+
+let wrapper;
 
 //Need at least 1 test to pass Travis CI
-describe ("<ArtistsContainer />", () => {
-  it ("creates component without crashing", () => {
-    const artists = <Artists/>;
+describe("<ArtistsContainer />", () => {
+  beforeEach(() => {
+    wrapper = shallow( < Artists / > )
+  })
+
+  it("contains the text 'ArtistsContainer'", () => {
+    expect(wrapper.find('div').text()).toEqual('ArtistsContainer');
   });
```

Comment:
Please remove beforeEach.

Modify to `const artists = shallow( < Artists/> )` and move down to just above the expectation on line 16.
You'll need to change `wrapper` to `artists `.